cves/CVE-2007-0404.yml

```diff
@@ -49,27 +49,39 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A script for compliling messages was made vulnerable because the inputs
+  submitted were not being checked to make sure that they were not malicious in nature.
+  The script job was to make OS level commands and if the inputs were not checked correctly
+  changes could to be made to the system if a targetted input is submitted as an arguememnt or in the .po file
+  with commands. If some of these commands are exectuted hackers can take over the system by giving themselves
+  access to everything possible.
+
+  The fix to this vulenrability was passing the input into a system tool that does this sanitizing
+  and removes any possibly malicious strings or characters in the command like ";" and "$"
 bounty_instructions: |
   If you came across any indications that a bounty was paid out for this
   vulnerability, fill it out here. Or correct it if the information already here
   was wrong. Otherwise, leave it blank.
-bounty:
+bounty: 
   amt:
   announced:
   url:
 reviews: []
-bugs: []
-repo:
+bugs: [407519]
```

Comment:
I believe this bug # is from Debian, not from Django itself.

Reply:
Yes, you're correct, I couldn't find a bug on the Django end as it was part of a release and didn't point to any bug fix. Will fix accordingly